metricflow/test/fixtures/manifest_fixtures.py

```diff
@@ -0,0 +1,201 @@
+            semantic_manifest_lookup=self.semantic_manifest_lookup,
+            node_output_resolver=node_output_resolver,
+            column_association_resolver=self.column_association_resolver,
+        )
+
+    @staticmethod
+    def _data_set_to_read_nodes(
+        data_sets: OrderedDict[str, SemanticModelDataSet]
+    ) -> OrderedDict[str, ReadSqlSourceNode]:
+        """Return a mapping from the name of the semantic model to the dataflow plan node that reads from it."""
+        # Moved from model_fixtures.py.
+        return_dict: OrderedDict[str, ReadSqlSourceNode] = OrderedDict()
+        for semantic_model_name, data_set in data_sets.items():
+            return_dict[semantic_model_name] = ReadSqlSourceNode(data_set)
+            logger.debug(
+                f"For semantic model {semantic_model_name}, creating node_id {return_dict[semantic_model_name].node_id}"
+            )
+
+        return return_dict
+
+    @staticmethod
+    def _data_set_to_source_nodes(
+        semantic_manifest_lookup: SemanticManifestLookup, data_sets: OrderedDict[str, SemanticModelDataSet]
+    ) -> Sequence[BaseOutput]:
+        # Moved from model_fixtures.py.
+        source_node_builder = SourceNodeBuilder(semantic_manifest_lookup)
+        return source_node_builder.create_from_data_sets(list(data_sets.values()))
+
+    @staticmethod
+    def _build_time_spine_source_node(
+        semantic_manifest_lookup: SemanticManifestLookup,
+    ) -> MetricTimeDimensionTransformNode:
+        # Moved from model_fixtures.py.
+        return SourceNodeBuilder.build_time_spine_source_node(
+            time_spine_source=semantic_manifest_lookup.time_spine_source,
+            data_set_converter=SemanticModelToDataSetConverter(
+                column_association_resolver=DunderColumnAssociationResolver(semantic_manifest_lookup)
+            ),
+        )
+
+    @staticmethod
+    def _create_data_sets(
+        multihop_semantic_manifest_lookup: SemanticManifestLookup,
+    ) -> OrderedDict[str, SemanticModelDataSet]:
+        """Convert the SemanticModels in the model to SqlDataSets.
+
+        Key is the name of the semantic model, value is the associated data set.
+        """
+        # Moved from model_fixtures.py.
+
+        # Use ordered dict and sort by name to get consistency when running tests.
+        data_sets = OrderedDict()
+        semantic_models: Sequence[SemanticModel] = multihop_semantic_manifest_lookup.semantic_manifest.semantic_models
+        semantic_models = sorted(semantic_models, key=lambda x: x.name)
+
+        converter = SemanticModelToDataSetConverter(
+            column_association_resolver=DunderColumnAssociationResolver(multihop_semantic_manifest_lookup)
+        )
+
+        for semantic_model in semantic_models:
+            data_sets[semantic_model.name] = converter.create_sql_source_data_set(semantic_model)
+
+        return data_sets
+
+
+@pytest.fixture(scope="session")
```

Comment:
Replied as a top-level comment.

> Also, since this is apparently not part of the stack with https://github.com/dbt-labs/metricflow/pull/1015 (or maybe that one was moved on top of this somewhere?) it's kind of hard for me to reason about how those two things will work together.

Should have outlined this better, but yes, this PR is part of the same stack and is under #1015.